concurrent_test.go

```diff
@@ -265,6 +265,24 @@ func runWorkers(t *testing.T,
 	return rs
 }
 
+func runWorker(t *testing.T, w worker, errCh chan error) (historyRecords, error) {
+	rs, err := w.run()
+	if terr := validateIncrementalTxid(rs, w.operationType()); terr != nil {
+		txidErr := fmt.Errorf("[%s]: %w", w.operationType(), terr)
+		t.Error(txidErr)
+		if err == nil {
+			errCh <- txidErr
+			return rs, txidErr
+		}
+	}
```

Comment:
Can you separate err and terr handling. It's unneseserly complicated.

Reply:
updated & thx